btr-web/btr-main-app/components/individual-person/AddNew.vue

```diff
@@ -63,10 +63,11 @@
         <p>
           {{ $t('texts.beneficialOwnershipAssessmentText1') }}
         </p>
-        <br>
+        <!-- NB: might be added back in later -->
```

Comment:
if chance of reading is not high, I would remove it altogether :)

Reply:
yeah sounds good